pihole_adlist_tool

```diff
@@ -350,12 +351,22 @@ if [ "$UNIQUE" -eq 1 ];
         echo -e "  [i]  UNIQUE: Not shown"
 fi
 
+# do not RegEx when running in docker container:
+if [ "$PIHOLE_DOCKER" = 0 ]; then
+  REGEX_MODE=1
```

Comment:
This does not work. It will always enable regex analysis in non-docker environments, regardless of the regex flag set or not. You need to check for `PIHOLE_DOCKER=1` and then set `REGEX_MODE=0`

Reply:
I must have had some weird knots in my brain. You're absolutely right 👍🏻